common/buildings/00_duchy_capital_buildings.txt

```diff
@@ -341,6 +341,9 @@ siege_works_02 = {
 	character_modifier = {
```

Comment:
Not all `light_cavalry_` modifiers were added to light aerial:
`parthian_tactics_perk`, `pastures_07`, `pastures_08`

The same with other units:
`silvermoon`, `feast_archers_insight_modifier`, `draktharon`, `lulkail`, `feast_heavy_infantry_insight_modifier`.
Are these intended?

That's how I find them.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/103769088-1f1aa880-503d-11eb-833a-e62740afd358.png)

</details>